packages/a11y-testing/src/definitions/Button/buttonBehaviorDefinition.ts

```diff
@@ -24,19 +24,15 @@ export const buttonBehaviorDefinition: Rule[] = [
     .doesNotHaveAttribute('disabled')
     .description(`if element is NOT native 'button'.`),
   BehaviorRule.root()
```

Comment:
split into two tests were done because of documentation reason 
before:
![image](https://user-images.githubusercontent.com/22620150/106279521-d372ad80-623c-11eb-84bb-14b21d9192be.png)

after: 
![image](https://user-images.githubusercontent.com/22620150/106279417-b2aa5800-623c-11eb-968a-b9a0d7ff2c5d.png)